src/main/java/seedu/duke/Storage.java

```diff
@@ -0,0 +1,53 @@
+package seedu.duke;
+
+import seedu.duke.task.Task;
+import java.io.IOException;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.ObjectInputStream;
+import java.io.ObjectOutputStream;
+import java.util.ArrayList;
+
+public class Storage {
+    String filepath;
+
+    public Storage(String filepath) {
+        this.filepath = filepath;
+    }
+
+    public ArrayList<Task> load() throws DukeException {
```

Comment:
Perhaps a better name would be "loadFile", which may help differentiate the method from similar future methods!